lib/galaxy/selenium/smart_components.py

```diff
@@ -98,3 +98,11 @@ def has_class(self, class_name):
 
     def wait_for_and_send_keys(self, text):
         self.wait_for_visible().send_keys(text)
+
+    def hack_js_click(self):
+        """Hack to force click at JavaScript level (bypassing screen interactions).
+
+        This should generally be avoided.
+        """
+        elem = self._has_driver.driver.find_element(*self._target.element_locator)
+        self._has_driver.driver.execute_script("arguments[0].click();", elem)
```

Comment:
Why do you need this? If it's a pseudo element, we have a very good way of `move_to_element_with_offset.click().perform()`

Reply:
I didn't work for me with this widget 😢 - this is why the tests are broken. I don't have an explanation.

Reply:
I tried that now... Why do you want to click exactly on `#prevent-job-dispatching`?
if you want to lock that thing, you can just do it on `job_lock_label`

Reply:
If this is a literal checkbox there was an issue with a chrome update that changed the way click() was executed; let me see if I can dig up a commit.